### ISSUE TITLE
playlist / queue fall out bugs + feedback

### ISSUE THREAD

**Issue by tzarebczan** [OdyseeTeam/odysee-frontend#1836]
- [ ] remove queue bottom bar on mobile refresh or when 0/0? 
 - [ ] Watch Later's "Start Playing" problems.
Clear your Watch Later.
Refresh,
Add a video into an existing private list from a homepage tile.
:x: Click "Start Playing" in the toast --> nothing happened.
Refresh
"Add to queue" on any video tile, then close the player.
:x: "Watch later" on any video tile --> it included the previous item from the Q, and now my Watch Later has 2 items  ... we only clicked it once.
[ ] --non-blocker-- Minor reflow issues when changing dropdown value in medium layout 
![image](https://user-images.githubusercontent.com/8120721/178752679-ec86873a-84a7-48fb-b0a2-04c1e780e931.png)

**Comment by infinite-persistence** [comment]
- [ ] Playlist doesn't play if first item (maybe also affects nth item) is DMCA'd

I manually copied the large playlist from Tom's wallet, which included `lbry://The-Matrix-Resurrections-(2021)#bb3cac2a46f8962c9d0fc84c6bf893711264c014` as the first entry.  When hitting "Play All", it showed the DMCA page rather than the playlist + skipping through the invalid item.

**Comment by infinite-persistence** [comment]
- [ ] It's possible to accidentally include livestreams in the Q, causing odd behavior.
  - Add a regular video into the Q.
  - Open and play a livestream. It will close the Q player and start playing the livestream.
  - Go back to homepage. Noticed that the livestream actually added into the Q.
    - :x: The "Prev" button does not work.  The 
    - :x: If you add another regular video in the Q, the "Next" button doesn't work while the livestream in playing.

Wanted to say "how about just allow livestreams in the Q and eventually playlists", but seems like there are more things to handle if we want to go that path.

**Comment by infinite-persistence** [comment]
- [ ] :arrow_down: Handle super-long playlist names
  - [ ] In list (maybe just truncate and show full name in tooltip)
  - [ ] In "Add to xxx" context menu ... it currently overflows the screen.

**Comment by infinite-persistence** [comment]
Unfinished business from the PR:
- [ ] [Can't play directly from toast if there's only one entry in the list](https://github.com/OdyseeTeam/odysee-frontend/pull/1604#discussion_r919915581)
- [ ] [Tip/Boost button not updated correctly](https://github.com/OdyseeTeam/odysee-frontend/pull/1604#discussion_r919919847)
- [ ] One more open comment in the PR.

**Comment by infinite-persistence** [comment]
Yeah, quite close to being fully functional (just the "next/prev" doesn't work as expected sometimes).  Feels like a useful feature to Q up.

**Comment by infinite-persistence** [comment]
- [ ] Show claim details, or at least bring back Tags?

The removal of claim details probably stemmed from the recent discussion to remove it (although I'd argue it still contains useful info like Languages), but tags seems still useful to retain?

<img width="827" alt="image" src="https://user-images.githubusercontent.com/64950861/179235594-c524e1e2-7049-44dc-971d-7a9a9020d135.png">
